src/cli/manager.rs

```diff
@@ -51,6 +51,15 @@ pub fn mavlink_connection_string() -> Option<&'static str> {
     return MANAGER.as_ref().clap_matches.value_of("mavlink");
 }
 
+pub(crate) fn log_path() -> String {
+    MANAGER
+        .as_ref()
+        .clap_matches
+        .value_of("log-path")
+        .expect("Clap arg \"log-path\" should always be \"Some(_)\" because of the default value.")
```

Comment:
Ok, the expect makes sense here, but that's kinda of unnecessary.
I would recommend for us to move to [clap derive](https://docs.rs/clap/latest/clap/) in another PR to avoid such poor consequences of the current architecture of clap.

Reply:
We also should also update to a more recent clap :+1: